server/src/service.rs

```diff
@@ -114,16 +114,64 @@ impl Service {
     }
 
     #[zbus(out_args("collection"))]
-    pub async fn read_alias(&self, _name: &str) -> Result<ObjectPath, ServiceError> {
-        todo!()
+    pub async fn read_alias(
+        &self,
+        name: &str,
+        #[zbus(object_server)] object_server: &zbus::ObjectServer,
+    ) -> Result<OwnedObjectPath, ServiceError> {
+        let collections = self.collections.lock().await;
+
+        for collection in collections.iter() {
+            let collection_ifce_ref = object_server.interface::<_, Collection>(collection).await?;
+            let collection = collection_ifce_ref.get_mut().await;
+
+            if name == "login" {
+                // To support login alias lookup
+                if collection.label().await == name {
```

Comment:
```suggestion
            // To support login alias lookup
            if name == "login" && collection.label().await == name {
```

Reply:
I'm getting the following clippy warning with this approach.
`warning: this `if` has identical blocks`

Reply:
yes that makes sense because the else if is returning the exact same error. So you should write

`if (name == "login" && collection.label().await == name ) || collection.alias().await == name`

Reply:
and also, the comment doesn't make sense anymore :)

Reply:
why do you have to check the label for a login collection instead of it alias? because the alias is set to default? then check if the alias is default?

Reply:
Yeah, I'll take another look.

Reply:
I have double checked this. Seahorse still works without this ("login" alias lookup) change. So, I have removed this.